<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pilotage frein manuel - Banc d'essais moteur</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox {
    background-color: #ffffff;
    color: #333333;
    border: 1px solid #cccccc;
    border-radius: 5px;
    padding: 4px 26px 4px 8px;
}

QComboBox:hover {
    border: 1px solid #66aaff;
}

QComboBox:focus {
    border: 1px solid #3399ff;
    background-color: #f0f8ff;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 24px;
    border-left: 1px solid #cccccc;
    background-color: #f0f0f0;
    border-top-right-radius: 5px;
    border-bottom-right-radius: 5px;
}

QComboBox::down-arrow {
    image: none; /* Utilise la flèche par défaut */
    width: 12px;
    height: 12px;
}

QComboBox QAbstractItemView {
    background-color: #ffffff;
    border: 1px solid #cccccc;
    selection-background-color: #3399ff;
    selection-color: white;
    color: #333333;
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #cce6ff;     /* Bleu très pâle */
    color: #003366;                /* Bleu foncé pour un bon contraste */
    border: 1px solid #66b3ff;     /* Bordure bleu clair */
    border-radius: 4px;
    padding: 5px;
}
</string>
   </property>
   <property name="text">
    <string>Choisir la consigne de frein </string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tE_valeur_temps_reel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>291</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
    background-color: white; /* fond blanc */
    color: #2a2d34;
    border: 2px solid #5a8dee;
    border-radius: 6px;
    padding: 6px;
}

QTextEdit:hover {
    border-color: #7faaff;
    /* pas de changement de fond au hover */
}

QTextEdit:focus {
    border-color: #a3baff;
    /* pas de changement de fond au focus */
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #e6f2ff;    /* Bleu très clair, style info */
    color: #0d47a1;               /* Bleu foncé, lisible et sérieux */
    border: 1px solid #90caf9;    /* Bordure douce et informative */
    border-radius: 4px;
    padding: 6px;
    font-size: 14px;
}</string>
   </property>
   <property name="text">
    <string>Valeurs en temps réel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cB_choix_enregistrement">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>121</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #dfe6e9;
    border: 1px solid #b2bec3;
    border-radius: 5px;
    padding: 6px 30px 6px 6px;
    font: 12px;
    color: #2d3436;
}

QComboBox:hover {
    background-color: #b2bec3;
}

QComboBox:pressed {
    background-color: #636e72;
    color: #ffffff;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 20px;
    border-left: 1px solid #b2bec3;
}

QComboBox QAbstractItemView {
    background-color: #dfe6e9;
    border: 1px solid #b2bec3;
    selection-background-color: #636e72;
    selection-color: #ffffff;
}
</string>
   </property>
   <item>
    <property name="text">
     <string>Choisir la Méthode d'Enregistrement des Données</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>En Continu</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Point par Point</string>
    </property>
   </item>
  </widget>
  <widget class="QGroupBox" name="gB_pilotage">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>281</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 2px solid qlineargradient(
        spread:pad, x1:0, y1:0, x2:1, y2:0,
        stop:0 #6dd5fa, stop:1 #2980b9
    );
    border-radius: 12px;
    margin-top: 25px;
    background-color: #ffffff;
    color: #1a1a1a;
    padding: 15px;
    font-family: &quot;Segoe UI&quot;, sans-serif;
    font-size: 14px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    background-color: transparent;
    color: #2c3e50;  /* Bleu-gris foncé, élégant et lisible */
    font-size: 16px;
    font-weight: bold;
    padding: 0 10px;
    text-transform: uppercase;
    letter-spacing: 1px;
}
</string>
   </property>
   <property name="title">
    <string>Pilotage du frein</string>
   </property>
   <widget class="QPushButton" name="pb_arret_frein">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #dfe6e9;
    border: 1px solid #b2bec3;
    border-radius: 5px;
    color: #2d3436;
    padding: 6px 12px;
    font: 13px;
}

QPushButton:hover {
    background-color: #b2bec3;
}

QPushButton:pressed {
    background-color: #636e72;
    color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Arrêt</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_valeurFrein">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSlider {
    background: transparent;
}

QSlider::groove:horizontal {
    height: 8px;
    background: #E0E0E0;
    border-radius: 4px;
}

QSlider::handle:horizontal {
    background: #4A90E2;
    border: 2px solid #357ABD;
    width: 16px;
    height: 16px;
    margin: -4px 0;
    border-radius: 8px;
}

QSlider::sub-page:horizontal {
    background: #4CAF50;
    border-radius: 4px;
}

QSlider::add-page:horizontal {
    background: #E0E0E0;
    border-radius: 4px;
}
</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_valider_consigne_frein">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #dfe6e9;
    border: 1px solid #b2bec3;
    border-radius: 5px;
    color: #2d3436;
    padding: 6px 12px;
    font: 13px;
}

QPushButton:hover {
    background-color: #b2bec3;
}

QPushButton:pressed {
    background-color: #636e72;
    color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sB_valeurFrein">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
    background-color: #e8eaed;
    color: #2a2d34;
    border: 2px solid #5a8dee;
    border-radius: 6px;
    padding: 4px 10px;
    font: 11px;
}

QSpinBox:hover {
    border-color: #7faaff;
    background-color: #f4f6fa;
}

QSpinBox:focus {
    border-color: #a3baff;
    background-color: #e1e6f7;
}

QSpinBox::up-button, QSpinBox::down-button {
    background-color: #c8d1f0;
    border: none;
    width: 20px;
}

QSpinBox::up-button:hover, QSpinBox::down-button:hover {
    background-color: #7faaff;
}

QSpinBox::up-arrow {
    image: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIGhlaWdodD0iOCIgdmlld0JveD0iMCAwIDE2IDgiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBvbHlnb24gcG9pbnRzPSIwLDggOCwwIDE2LDgiIGZpbGw9IiMwMDhmZmYiLz48L3N2Zz4=);
    width: 16px;
    height: 8px;
}

QSpinBox::down-arrow {
    image: url(data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTYiIGhlaWdodD0iOCIgdmlld0JveD0iMCAwIDE2IDgiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBvbHlnb24gcG9pbnRzPSIwLDAgOCw4IDE2LDAiIGZpbGw9IiMwMDhmZmYiLz48L3N2Zz4=);
    width: 16px;
    height: 8px;
}
</string>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_marche_frein">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #dfe6e9;
    border: 1px solid #b2bec3;
    border-radius: 5px;
    color: #2d3436;
    padding: 6px 12px;
    font: 13px;
}

QPushButton:hover {
    background-color: #b2bec3;
}

QPushButton:pressed {
    background-color: #636e72;
    color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Marche</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pb_valider_enregistrement">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #dfe6e9;
    border: 1px solid #b2bec3;
    border-radius: 5px;
    color: #2d3436;
    padding: 6px 12px;
    font: 13px;
}

QPushButton:hover {
    background-color: #b2bec3;
}

QPushButton:pressed {
    background-color: #636e72;
    color: #ffffff;
}</string>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_retour_pilotage_frein">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>540</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
